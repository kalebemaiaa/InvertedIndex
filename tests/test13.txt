QueryOr(QueryAndL(QueryOr(QueryWord("presidente"), QueryWord("primeiro-ministro")), QueryAnd(QueryWord("diplomata"), QueryOr(QueryWord("embaixador"), QueryWord("secretário")))), QueryAndL(QueryOr(QueryWord("deputado"), QueryWord("senador")), QueryAnd(QueryWord("prefeito"), QueryOr(QueryWord("governador"), QueryWord("assessor")))))
11126.raw
3177.raw
11438.raw
5864.raw
728.raw
6059.raw
2385.raw
514.raw
11294.raw
3578.raw
4636.raw
1196.raw
1096.raw
1675.raw
509.raw
3083.raw
3095.raw
3723.raw
3713.raw
4364.raw
11854.raw
2536.raw
2602.raw
861.raw
1937.raw
2710.raw
2869.raw
559.raw
2347.raw
4879.raw
4274.raw
11056.raw
5703.raw
4521.raw
4644.raw
232.raw
11156.raw
2282.raw
11370.raw
3976.raw
4875.raw
2704.raw
1947.raw
1706.raw
5209.raw
6164.raw
4712.raw
7790.raw
4310.raw
6323.raw
11792.raw
231.raw
11271.raw
4161.raw
4766.raw
11724.raw
852.raw
2431.raw
2139.raw
4709.raw
5021.raw
4755.raw
2744.raw
11626.raw
4253.raw
5099.raw
886.raw
11298.raw
2134.raw
3983.raw
2630.raw
4615.raw
5721.raw
2273.raw
5880.raw
11617.raw
4813.raw
5054.raw
5997.raw
904.raw
5321.raw
4926.raw
818.raw
3741.raw
11362.raw
4635.raw
4816.raw
2977.raw
3606.raw
2520.raw
10802.raw
2526.raw
2129.raw
12017.raw
934.raw
1425.raw
2372.raw
909.raw
5527.raw
3441.raw
1259.raw
1516.raw
11695.raw
11826.raw
1124.raw
2032.raw
6110.raw
5692.raw
11420.raw
360.raw
5562.raw
3840.raw
6041.raw
532.raw
6400.raw
3742.raw
1095.raw
4753.raw
2335.raw
6347.raw
5528.raw
4426.raw
11642.raw
427.raw
2631.raw
11820.raw
11880.raw
12022.raw
6013.raw
4981.raw
5563.raw
4476.raw
585.raw
753.raw
11906.raw
5190.raw
3082.raw
10993.raw
2884.raw
2539.raw
486.raw
5179.raw
1599.raw
313.raw
437.raw
1953.raw
5422.raw
5394.raw
566.raw
3165.raw
1571.raw
3799.raw
1610.raw
2133.raw
5847.raw
5664.raw
3314.raw
763.raw
1274.raw
2566.raw
4874.raw
4067.raw
3103.raw
3260.raw
5551.raw
1929.raw
5616.raw
2670.raw
3565.raw
375.raw
2941.raw
11879.raw
5907.raw
1315.raw
5792.raw
4326.raw
11701.raw
6321.raw
1692.raw
11055.raw
4598.raw
34.raw
633.raw
2590.raw
3950.raw
8831.raw
11431.raw
5117.raw
6121.raw
3663.raw
6160.raw
11595.raw
1954.raw
11291.raw
5504.raw
11866.raw
1860.raw
3168.raw
773.raw
3643.raw
