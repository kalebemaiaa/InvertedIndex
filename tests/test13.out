728
886
1274
1425
1571
1610
1692
1706
1929
1947
1954
2129
2133
2273
2520
2536
2602
2630
2670
2710
3165
3168
3314
3663
3799
4067
4326
4755
4816
5209
5562
5563
5792
10802
11056
11126
11695
11701
11906
